specification/servicenetworking/ServiceNetworking.Management/examples/2024-05-01-preview/SecurityPolicyGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "Get SecurityPolicy",
+  "operationId": "SecurityPoliciesInterface_Get",
+  "parameters": {
+    "api-version": "2024-05-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "trafficControllerName": "tc1",
+    "securityPolicyName": "sp1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "waf-0",
+        "type": "Microsoft.ServiceNetworking/trafficControllers/securityPolicies",
+        "id": "/subscriptions/subid/resourcegroups/rg1/providers/Microsoft.ServiceNetworking/trafficControllers/sample-tc/securityPolicies/waf-0",
```

Comment:
for a realistic example, the resource id should match the input parameters

Reply:
agreed